plugin/core/windows.py

```diff
@@ -435,6 +449,11 @@ def _start_client(self, config: ClientConfig) -> None:
         if session:
             debug("window {} added session {}".format(self._window.id(), config.name))
             self._sessions[config.name] = session
+            transport = session.client.transport
+            if isinstance(transport, StdioTransport) and settings.log_stderr:
```

Comment:
Fishing up a process after the client is created is not the easiest as we've tried to abstract it from much of the code outside session :) 
Perhaps we should just expose an `Optional` process somewhere on Client and/or Session?

Otherwise, can `create_session` help? It is the only place that definitely knows if a process is created.

Reply:
Fixed by passing a callback along in `create_session`.